<launch>
    
    <!--- Sim Time -->
    <param name="/use_sim_time" value="false" />

    <!--- Run Rviz-->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lili_om_rot)/config/test.rviz" />

    <!-- Parameters -->
    <rosparam file = "$(find lili_om_rot)/config/config_fr_iosb.yaml"/>
    
    <node pkg="tf" type="static_transform_publisher" name="lili_om_rot" args="0 0 0 0 0 0 base_link lili_om_rot 50"/>

    <!--- lili_om_rot -->
    <node pkg="lili_om_rot" type="Preprocessing" name="Preprocessing" output="screen"/>
    <node pkg="lili_om_rot" type="LidarOdometry" name="LidarOdometry" output="screen"/>
    <node pkg="lili_om_rot" type="BackendFusion" name="BackendFusion" output="screen"/>

</launch>
